capstone/backend/src/main/java/com/google/sps/spanner/UsersSpanner.java

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package com.example.spanner;
```

Comment:
Change the package name to package com.java.spanner, and move the file to `src/main/java/com/google/spanner/CreateSpannerTables.java` or something. This file just contain the User tables, so `UsersSpanner.java` is not a good name.